templates/sshd.custom.erb

```diff
@@ -0,0 +1,15 @@
+# This file is being maintained by Puppet.
```

Comment:
True if the sequence does matter here. Looking at the short result of the existing tests I would recommend to use a variable instead of adding a a fixture file. Like here [1].

[1] https://github.com/ghoneycutt/puppet-module-bind/blob/master/spec/classes/init_spec.rb#L127-L149